MaterialLib/Fluid/Viscosity/TemperatureDependentViscosity.h

```diff
@@ -0,0 +1,97 @@
+/**
+ *  \brief A linear temperature dependent viscosity model.
+ *
+ *  \copyright
+ *   Copyright (c) 2012-2016, OpenGeoSys Community (http://www.opengeosys.org)
+ *              Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ * \file:   TemperatureDependentViscosity.h
+ *
+ *  Created on August 10, 2016, 10:45 AM
+ */
+
+#ifndef TEMPERATUREDEPENDENTVISCOSITY_H
+#define TEMPERATUREDEPENDENTVISCOSITY_H
+
+#include <string>
+#include <cmath>
+
+#include "BaseLib/ConfigTree.h"
+
+#include "MaterialLib/Fluid/FluidProperty.h"
+
+namespace MaterialLib
+{
+namespace Fluid
+{
+/**
+ *  Class of a temperature dependent viscosity model
+ *   \f[
+ *        \mu = \mu_0\, exp(\frac{T-T_c}{T_v})
+ *   where
+ *    \f{eqnarray*}{
+ *       &\mu_0:&  \mbox{reference viscosity,}\\
+ *       &T:&      \mbox{temperature,}\\
+ *       &T_c:&    \mbox{parameter,}\\
+ *       &T_v:&    \mbox{parameter,}\\
```

Comment:
What are the T_c and T_v parameters.
The formula is different from implementation. The exponent is negative in the latter.